jenkins_jobs/upload_iso/script.sh

```diff
@@ -6,6 +6,7 @@ ISO_DIR_RSYNC_URL="${BUILD_DIR_RSYNC_URL}/iso"
 rsync_upload() {
     rsync -e "ssh -i ${HOME}/.ssh/${UPLOAD_SERVER_USER_NAME}_id_rsa" \
               --verbose --compress --stats --times --chmod=a+rwx,g+wx,o- \
+              --ignore-existing --itemize-changes \
```

Comment:
I'm sorry for suggesting the use of this parameter, but since we already have `--verbose`, `--itemize-changes` does not add any useful information. The files that are sent are already listed, and the extra update information given by `--itemize-changes` will never exist, since we are ignoring existent files.